lib/media/drm_engine.js

```diff
@@ -1254,28 +1253,6 @@ shaka.media.DrmEngine = class {
     };
     this.activeSessions_.set(session, metadata);
 
-    /**
-     * initDataTransform is only necessary when using legacy protection
```

Comment:
contentID is only needed on legacy APIs?

Reply:
Correct